src/main/kotlin/com/example/passmanager/domain/MongoPassOwner.kt

```diff
@@ -12,9 +14,12 @@ data class MongoPassOwner(
     @Id val id: ObjectId?,
     val firstName: String?,
     val lastName: String?,
+    @Indexed(name = "phone_number_index", unique = true)
```

Comment:
Instead of usage @Indexed or corresponding class level annotation use Mongock migration and create indexes only in @ChangeUnit.

@Indexed will always recreate indexes even after you drop it by migration at some day